src/containerapp-preview/azext_containerapp_preview/_help.py

```diff
@@ -0,0 +1,64 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+helps['containerapp create'] = """
+    type: command
+    short-summary: Create a container app.
+    examples:
+    - name: Create a container app and retrieve its fully qualified domain name.
+      text: |
+          az containerapp create -n MyContainerapp -g MyResourceGroup \\
+              --image myregistry.azurecr.io/my-app:v1.0 --environment MyContainerappEnv \\
+              --ingress external --target-port 80 \\
+              --registry-server myregistry.azurecr.io --registry-username myregistry --registry-password $REGISTRY_PASSWORD \\
+              --query properties.configuration.ingress.fqdn
+    - name: Create a container app with resource requirements and replica count limits.
+      text: |
+          az containerapp create -n MyContainerapp -g MyResourceGroup \\
+              --image nginx --environment MyContainerappEnv \\
+              --cpu 0.5 --memory 1.0Gi \\
+              --min-replicas 4 --max-replicas 8
+    - name: Create a container app with secrets and environment variables.
+      text: |
+          az containerapp create -n MyContainerapp -g MyResourceGroup \\
+              --image my-app:v1.0 --environment MyContainerappEnv \\
+              --secrets mysecret=secretvalue1 anothersecret="secret value 2" \\
+              --env-vars GREETING="Hello, world" SECRETENV=secretref:anothersecret
+    - name: Create a container app using a YAML configuration. Example YAML configuration - https://aka.ms/azure-container-apps-yaml
+      text: |
+          az containerapp create -n MyContainerapp -g MyResourceGroup \\
+              --environment MyContainerappEnv \\
+              --yaml "path/to/yaml/file.yml"
+    - name: Create a container app with an http scale rule
+      text: |
+          az containerapp create -n myapp -g mygroup --environment myenv --image nginx \\
+              --scale-rule-name my-http-rule \\
+              --scale-rule-http-concurrency 50
+    - name: Create a container app with a custom scale rule
+      text: |
+          az containerapp create -n MyContainerapp -g MyResourceGroup \\
```

Comment:
consider reusing help in GA version to avoid dup code

Reply:
[Task 24502214](https://msazure.visualstudio.com/Antares/_workitems/edit/24502214): containerapp-preview helps.py consider reusing help in GA version to avoid dup code